### ISSUE TITLE
Add coin toss and/or customizable dice values

### ISSUE THREAD

**Issue by Porkepix** [mueller-ma/TabletopTools#283]
**Is your feature request related to a problem? Please describe.**
A coin toss is something that can oftentimes be useful in many games.
One could also consider that it's a "custom die" of value 2; it could be possible to be able to throw dices of customized value, and 2 would be an option.
This goes a little bit along issue https://github.com/mueller-ma/TabletopTools/issues/151

Basic example of usage: pick who's the first player to go.

**Describe the solution you'd like**
A coin option & dices of customizable values.

**Describe alternatives you've considered**
Use dices of higher value and divide the result for example. This is still less handy.

**Comment by mueller-ma** [comment]
#151 was more complex than I originally thought, thus I didn't implement it yet. A coin toss would be a great addition, I'm open for PRs here ;)

As an alternative you can use the random list feature.

**Comment by Jenish-1235** [comment]
Hello there, I would like to contribute to this  issue during hacktoberfest '24. Could you let me know whether this issue is still pending or not?

**Comment by mueller-ma** [comment]
Yes, the coin feature is still pending and you can contribute it in oktober.

**Comment by Jenish-1235** [comment]
Sure thanks for your response, you assign it to you.
